components/src/core/components/CardTable/Cell/Actions.vue

```diff
@@ -54,10 +54,10 @@ export default defineComponent({
 
   props: {
     item: {
-      type: Object,
+      type: null,
       // not required, since actions evaluate from `this.header.cellConfig`
       required: false,
-      default: () => ({}),
+      default: () => undefined,
```

Comment:
do we need this line?
```suggestion
```

Reply:
due to lint rules it's necessary. if the prop is not required, default must be defined.